src/components/UniversalNavbarExpanded/constants.js

```diff
@@ -0,0 +1,191 @@
+import uuidv4 from 'uuid/v4'
+export const LINKS = {
+  // These are used e.g. in the logo, icons and bottom of mobile nav:
+  INDEX: { href: '/' },
+  TERM: {
+    href: '/term',
+    title: 'Check my price',
+  },
+  // TALK: {
+  //     id: uuidv4(),
+  //     href: '/',
+  //     title: 'Talk to us',
+  //   },
+  ACCOUNT: {
+    href: '/login/',
+    title: 'Account',
+  },
+  SEARCH: {
+    href: '/search/',
+    title: 'Search',
+  },
+
+  // These are the main top level link and their subnav items
+  NAVLINKS: [
```

Comment:
These really don’t belong in EDS I think. CMS should be passing these in. As it stands this component is a bit weak on reuse because if lack of dependency injection. Or are we in fact importing this file from the outside and passing these values in?

Reply:
Tamir brought up the same point. I'll see if I can fit this in tomorrow too, would be faster for future updates to these values if we are passing them in from CMS.